app/lib/server/server.go

```diff
@@ -144,10 +146,17 @@ func InteractiveConfig(binaryName string) {
 
 	// create the keys
 	privStr, pubStr := createKeyPair()
+	ips, err := net.LookupIP(publicAddress.Host())
+	log.ErrFatal(err)
+	cert, key, err := network.NewCertKey(network.NewTLSCert(big.NewInt(1), "ch", "epfl", "dedis",
```

Comment:
Well, the tests scripts are not the end goal, but rather the user don't you think ? It's our job to make the two work together. 
+ user input: the user is being asked explicitely for each each so it's kinda clear
+ arguments: well, if there were only 2 inputs why not, but there's more into a certificate so having a very loooooong line of options when generating the setup... I'm not sure. And we already do user input, don't you think it's strange to ask for options and inputs at the same time for different things ?